### PR TITLE
[DOCS] How to disable all logging

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/Logging/Configuration/Index.rst [hunk 1]
```diff
@@ -139,3 +139,26 @@ can be configured on a per-class and per-namespace basis with the sub-key
...
 :ref:`logging-processors`.
+
+..    _logging-configuration-disable:
+
+Disable all logging
+===================
+
+In some setups it is desirable to disable all logs and to only enable them on demand.
+You can disable all logs by unsetting :php:`$GLOBALS['TYPO3_CONF_VARS']['LOG']` at the 
+end of your :ref:`adiitional.php <typo3ConfVars-additional>`:
```
Comment (garvinhicking) [inline Documentation/ApiOverview/Logging/Configuration/Index.rst:150]:
```suggestion
end of your :ref:`additional.php <typo3ConfVars-additional>`:
```

File: Documentation/ApiOverview/Logging/Configuration/Index.rst [hunk 2]
```diff
@@ -139,3 +139,26 @@ can be configured on a per-class and per-namespace basis with the sub-key
...
+
+..  code-block:: php
+    :caption: config/system/additional.php | typo3conf/system/additional.php
+
+    // disable all logging
+    unset($GLOBALS['TYPO3_CONF_VARS']['LOG']);    
+
+You can then temporarily enable logging by commenting out this line:
```
Comment (garvinhicking) [inline Documentation/ApiOverview/Logging/Configuration/Index.rst:158]: I'm not sure the last paragraph about temporarily re-enabling this is needed, IMO it should be clear when you disable it with a line, that you can remove it...?

Comment (linawolf) [reply Documentation/ApiOverview/Logging/Configuration/Index.rst:158]: I think this demonstrates the advantage of a global unset compared to other methods of disabling logs: You can quickly enable and disable it. Also there is not harm in being explicit...?

Comment (garvinhicking) [reply Documentation/ApiOverview/Logging/Configuration/Index.rst:158]: Acknowledged

**Review by garvinhicking — APPROVED** [review]

**Pull request merged**